package.json

```diff
@@ -6,7 +6,7 @@
     "stylelint": "stylelint app/assets/stylesheets",
     "test": "jest",
     "build": "esbuild app/javascript/*.* --bundle --loader:.gif=dataurl --sourcemap --minify --outdir=app/assets/builds --public-path=assets --target=es6",
-    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --style=compressed --load-path=node_modules --quiet-deps --silence-deprecation slash-div --silence-deprecation import",
+    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --style=compressed --load-path=. --load-path=node_modules --quiet-deps --silence-deprecation slash-div --silence-deprecation import",
```

Comment:
Multiple load paths, would the root (`--load-path=.`) load path be sufficient/smart enough to look within the `node_modules folder`?

Reply:
Error: "Can't find stylesheet to import" when only the root path is loaded, so I guess we need both.